Adyen/Analytics/AnalyticsProvider/AnalyticsProvider.swift

```diff
@@ -12,11 +12,14 @@ public struct AnalyticsConfiguration {
 
     // MARK: - Properties
 
+    /// A Boolean value that determines whether analytics is enabled.
+    public var isEnabled = true
```

Comment:
didn't we agree not to have a kill switch like this one, since its complicated to keep all the switches in sync?